scripts/install

```diff
@@ -113,21 +123,28 @@ $final_path/.pythonz/bin/pythonz install 3.6.6
 # INSTALL YUNORUNNER DEPENDENCIES
 #=================================================
 
-( cd $final_path
-virtualenv -p $($final_path/.pythonz/bin/pythonz locate 3.6.6) ve3
-ve3/bin/pip3 install -r requirements.txt )
+pushd $final_path
```

Comment:
Why not keeping the sub shell ?

Reply:
I don't know, just more clear for me to read code

Reply:
Ok, no problem for me